README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/AndreaCrotti/code-maat.png)](https://travis-ci.org/AndreaCrotti/code-maat)
```

Comment:
Ah sorry there is clearly an issue here @adamtornhill since it's pointing to my account.

I think if you haven't already before you enable this you might need to enable your travis-ci account and add the repository in the enabled ones, then it will magically do all the builds and show the status as png.

So if  you agree on this change let me know and I'll change the name to yours.